worlds/ubp.sdf

```diff
@@ -0,0 +1,224 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sdf version="1.9">
+  <world name="ubp">
```

Comment:
What is ubp? Is there a more descriptive name you can use?

Reply:
yeah it is not a clear name so i think `campus` might be suitable and descriptive for this world model